packages/quantic/force-app/custom-token/main/default/objects/TokenSettings__c/TokenSettings__c.object-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
Add description on CustomObject def and CustomField def to explain what they are.

Reply:
- [ ] ToDo: Description on the custom settings and its fields